jest.setup.js

```diff
@@ -1,2 +1,3 @@
 /* global jest */
 jest.setTimeout(120000);
+jest.retryTimes(3, {logErrorsBeforeRetry: true})
```

Comment:
retry test 3 times on failure and log retries:
https://jestjs.io/docs/jest-object#jestretrytimesnumretries-options